e2e/tests/upgrades/upgrade_test.go

```diff
@@ -70,6 +73,8 @@ func (s *UpgradeTestSuite) UpgradeChain(ctx context.Context, chain *cosmos.Cosmo
 	err = chain.StartAllNodes(ctx)
 	s.Require().NoError(err, "error starting upgraded node(s)")
 
+	s.InitGRPCClients(chain)
```

Comment:
just a note for the future, it might make sense to bundle these into a `restartChain` function. If we ever have another function which stops a chain, we will need to realize to initialize the grpc clients again. I'm very fine with this for now